LFS-722: Fields at the bottom of the form are inaccessible

marta- opened the pull request:
Introduced a separate save button, outside of the pagination component, styled similarly to the main action buttons on Dashboard, Forms, Subjects.
![image](https://user-images.githubusercontent.com/651980/102018301-7da07f80-3d3a-11eb-8f80-e9a1a95acf3e.png)

Additionally, for LFS-783 (As a user filling out a form, I can see when the data was last saved in the current session improved feedback displayed to the user): when pressing the save button fails to save the data, display an error dialog which includes the time of the last save.
![image](https://user-images.githubusercontent.com/651980/102018243-37e3b700-3d3a-11eb-92d8-e79e938e666f.png)

marta- committed:
LFS-722: Fields at the bottom of the form are inaccessible
Introduced a separate save button, outside of the pagination component, styled similarly to the main action buttons on Dashboard, Forms, Subjects.

sdumitriu reviewed (approved):
I find the icon and the text in the button too close together.

sashaandjic reviewed (approved):
Agree with Sergiu. The icon and label appear to be too close.
![image](https://user-images.githubusercontent.com/23243244/102002914-40eb6e80-3ccf-11eb-88c6-1622df823c10.png)
![image](https://user-images.githubusercontent.com/23243244/102002918-45b02280-3ccf-11eb-9c0e-a6a3e3543ae9.png)

marta- committed:
LFS-722: Fields at the bottom of the form are inaccessible
Improved save button spacing

marta- committed:
[misc] Use theme color for from header background

marta- committed:
LFS-783: As a user filling out a form, I can see when the data was last saved in the current session
Improved feedback displayed to the user when pressing the save button fails to save the data

sashaandjic reviewed (approved):
much better 
Isn't the Save bttn kind of obsolete now that all changes are saved once user leaves the form?
![image](https://user-images.githubusercontent.com/23243244/102019840-aaa56000-3d43-11eb-99eb-243ac6a794d8.png)![image](https://user-images.githubusercontent.com/23243244/102019827-9c574400-3d43-11eb-9de0-303a00de2f3b.png)
